src/Component/ScaleDenominator/MaxScaleDenominator.tsx

```diff
@@ -51,8 +58,9 @@ export interface MaxScaleDenominatorProps extends Partial<MaxScaleDenominatorDef
  * Input field for the maximum scale of a rule.
  */
 export const MaxScaleDenominator: React.FC<MaxScaleDenominatorProps> = ({
-  label = 'Max. Scale',
-  placeholder = 'Enter max. Scale (Optional)',
+  placeholder,
+  locale = en_US.ScaleDenominator,
+  label = locale.maxScaleDenominatorLabelText,
```

Comment:
I think the extra `label` prop should be removed here and we should use `locale.maxScaleDenominatorLabelText` in the JSX element instead.